cli/command-reference/roxctl-generate.adoc

```diff
@@ -0,0 +1,39 @@
+[id="roxctl-generate"]
```

Comment:
Remove the `roxctl generate` part, this has been replaced with `roxctl netpol generate` and is here solely for backwards compatibility.